readme.md

```diff
@@ -0,0 +1,288 @@
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
```

Comment:
> Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

- [ ] You did great using the provided Microverse readme template. :+1: :+1: Looking at the readme file, it seems you have not customized it to your project. As the previous reviewer suggested and also According to Microverse the only optional section is the FAQ, you are meant to customize every other section in your readme file. :smile: :wave: I'll suggest you get the [template once more,](https://github.com/microverseinc/readme-template) make sure you do not remove any section. Then go through your readme and remove the placeholder logo and texts, and then fill the content of every section in the readme file :+1: :100: 

**THIS IS AN EXAMPLE OF HOW THE REAMDE SHOULD LOOK LIKE FOR THIS PROJECT. [LINK](https://github.com/enocol/HelloMicroverse/blob/newbranch/README.md)**

Reply:
Update the readme file as you suggested